src/utils/errors/account.ts

```diff
@@ -20,6 +20,9 @@ const messages = defineMessages({
   },
   tooSimilar: {
     defaultMessage: "These passwords are too similar"
+  },
+  unique: {
+    defaultMessage: "This needs to be unique"
```

Comment:
I'm not sure if this error should have generic error message 🤷‍♂️

Reply:
Do you think it should be "Name should be unique"?

Reply:
Depends what should be unique, "name" and "slug" would have different messages. It would be nice to know what field(s) could throw this error and make proper function for each field.

Reply:
It could be thrown only by name field so I don't think separate functions are needed. Change to 'Name needs to be unique'?